packages/blockchain-wallet-v4/src/types/__mocks__/wrapper.v4.json

```diff
@@ -0,0 +1,48 @@
+{
+  "sync_pubkeys": false,
+  "serverTime": 1537471942924,
+  "real_auth_type": 0,
+  "auth_type": 0,
+  "payload_checksum": "ca9317656c34fd5677eaf3d312e7663eefd479d1bcea915ee73c4dcaf7f7292e",
+  "version": 3,
+  "language": "en",
+  "war_checksum": "a5493d86b6ce640e",
+  "password": "Blockchain*",
+  "clientTimeDiff": 154,
+  "pbkdf2_iterations": 5000,
+  "wallet": {
+    "addresses": [],
+    "tx_notes": [],
+    "guid": "f2dab335-b5ab-40fc-9e1e-570779651501",
+    "tx_names": [],
+    "double_encryption": false,
+    "address_book": [],
+    "sharedKey": "9bf56d39-c60f-4b80-8c15-9fbe8ad537a3",
+    "hd_wallets": [
+      {
+        "seed_hex": "6a4d9524d413fdf69ca1b5664d1d6db0",
+        "passphrase": "",
+        "mnemonic_verified": false,
+        "default_account_idx": 0,,
```

Comment:
extra comma here

Reply:
Removing this file. I was trying to get something to work but it's no longer needed.